src/app/components/Menu/Menu.tsx

```diff
@@ -5,18 +5,51 @@ import './Menu.scss';
 import Button from '../Button/Button';
 import { useRouter } from 'next/navigation';
 
+interface Locale {
+  language: string;
+  tag: string;
+}
+
+const locale: Record<string, Locale> = {
+  english: {
+    language: 'English',
+    tag: 'en-US',
+  },
+  polish: {
+    language: 'Polish',
+    tag: 'pl-PL',
+  },
+  spanish: {
+    language: 'Spanish',
+    tag: 'es-ES',
+  },
+  french: {
+    language: 'French',
+    tag: 'fr-FR',
+  },
+};
+
```

Comment:
Te całe informacje odnosnie locale to w ogole trzeba gdzies przeniesc i to tutaj importować. 
Wydaje mi sie ze moze to poleciec do pliku i18nConfig